Gruntfile.js

```diff
@@ -3,6 +3,7 @@ module.exports = function(grunt) {
     grunt.initConfig({
         eslint: {
             options: {
+		    fix: true,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=4554164)